src/betterproto/plugin/parser.py

```diff
@@ -124,13 +124,12 @@ def generate_code(request: CodeGeneratorRequest) -> CodeGeneratorResponse:
 
     # Make each output directory a package with __init__ file
     init_files = {
-        directory.joinpath("__init__.py")
-        for path in output_paths
-        for directory in path.parents
+        directory / "__init__.py" for path in output_paths for directory in path.parents
```

Comment:
```suggestion
        directory / "__init__.py"
        for path in output_paths for directory in path.parents
        not directory.joinpath( "__init__.py").exists()
```

This might be better that way we also only add "Writing  " messages for the ones we actually write.